README.md

```diff
@@ -42,13 +41,13 @@ into your HTML pages:
 Add this to your `<head>` element:
 
 ```html
-<link rel="stylesheet" href="/path/to/your/assets/css/uswds-0.9.0.css">
+<link rel="stylesheet" href="/path/to/your/assets/css/uswds.css">
```

Comment:
I think `path/to/your/assets/` lets them decide enough about where it should go. but :thought_balloon: welcome. Let's settle on something, `lib` or `vendor`. 

It might make sense for it to look a little more like this.

```
<link rel="stylesheet" href="/path/to/your/assets/css/lib/uswds.css">
<script href="/path/to/your/assets/js/lib/uswds.js"></script
```